### PR TITLE
Gci generating certificate

### PR DESCRIPTION
**Pull request by IlyaMaier**
- Added "Add" button in Organization component
- Added validations to the template

[Task](https://codein.withgoogle.com/dashboard/task-instances/5399554875719680/)

Screenshots:
screenshot 40
screenshot 41
screenshot 42

### CODE REVIEW

**Comment by nikitadhiman** [comment]
https://github.com/JBossOutreach/certificate-generator-front/pull/101 you can. Refer this

**Comment by IlyaMaier** [comment]
Done.

**Comment by IlyaMaier** [comment]
I've changed also description.

File: src/app/organization/generate/generate.component.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<div class="card">
+  <form enctype="multipart/form-data" method="POST">
+
```
Comment (nikitadhiman) [inline src/app/organization/generate/generate.component.html]: method is not required, you may use template driven or reactive form feature of angular. They are very powerful form feature.

File: src/app/organization/generate/generate.component.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <div class="btn">
+      <input type="submit" value="Preview">
+    </div>
+
+    <div class="btn">
+      <input type="button" value="Publish">
+    </div>
+
+  </form>
```
Comment (nikitadhiman) [inline src/app/organization/generate/generate.component.html]: No validations? Please add validations for mandatory fields namely name, description etc

Comment (nikitadhiman) [reply src/app/organization/generate/generate.component.html]: Validation should be placed in component.ts files and not html files

**Pull request closed without merging**